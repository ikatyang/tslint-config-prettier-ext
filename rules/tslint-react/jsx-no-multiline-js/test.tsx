const printBadStr = <div>
{test ?
    "a!=b"
:
    "b!=a"
}
</div>;
