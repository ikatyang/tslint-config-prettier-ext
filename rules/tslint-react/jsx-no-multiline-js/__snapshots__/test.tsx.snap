// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (jsx-no-multiline-js) /rules/tslint-react/jsx-no-multiline-js/test.tsx[2, 1]: Multiline JS expressions inside JSX are forbidden

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const printBadStr = <div>
{test ?
    \\"a!=b\\"
:
    \\"b!=a\\"
}
</div>;

======
const printBadStr = (
  <div>
    {test ? \\"a!=b\\" : \\"b!=a\\"}
  </div>
);

>>>>>> after
"
`;
